protocol-units/types/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub use aptos_api::runtime::Apis;
+pub use aptos_crypto::hash::HashValue;
+pub use aptos_types::{
+	block_executor::partitioner::ExecutableBlock,
+	block_executor::partitioner::ExecutableTransactions,
+	transaction::signature_verified_transaction::SignatureVerifiedTransaction,
+	transaction::{SignedTransaction, Transaction},
+};
+
+pub mod executor;
+pub mod finality_mode;
+
+pub use executor::ExecutorOps;
+pub use finality_mode::FinalityMode;
```

Comment:
For now this looks like a grab bag of re-exports, and an abstract crate definition, for everything related to executors. Is it the sole purpose of the crate?

Reply:
The purpose of this crate is to allow consumers of the protocol units to include this types crate to get access to all the types that the interfaces or traits use. This way, we can add new protocol units and have the types shared.